src/ssl_stubs.c

```diff
@@ -831,7 +837,7 @@ CAMLprim value caml_alpn_select_cb(SSL *ssl,
   *out = String_val(selected_protocol);
   *outlen = len;
 
-  return SSL_TLSEXT_ERR_OK;
+  CAMLreturn(SSL_TLSEXT_ERR_OK);
```

Comment:
I believe this one is necessary for safety after the CAMLlocal above. You should also fix the return on line 826->832 I believe?